spec/controllers/answers_controller_spec.rb

```diff
@@ -46,6 +46,34 @@
     end
   end
 
+  describe "PATCH#approve" do
+    before { sign_in login_user }
+
+    it "質問者であれば、回答を承認できること" do
+      own_question = FactoryGirl.create(:question, user: login_user)
+      answer = FactoryGirl.create(:answer, question: own_question)
+      patch :approve, {
+        question_id: own_question.id,
+        id: answer.id
+      }
+      own_question.reload
+      expect(own_question.approved_answer).to eq answer
+    end
+
+    it "質問者ではない場合、回答を承認できないこと" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.